scripts/bootstrap_vm.sh

```diff
@@ -98,6 +98,30 @@ chown -R splunk:splunk $SPLUNK_HOME
 $SPLUNK_HOME/bin/splunk start
 }
 
+get_download_id () {
+    wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+    chmod +x ./jq
+    cp jq /usr/bin
+    json_data=$(curl -L --request GET --url 'https://www.tenable.com/downloads/api/v1/public/pages/nessus-agents/' --header 'Accept: aplication/json')
+    download_id=$(jq --arg desc "$1" '.. | select(.description? == $desc) | .id' <<< "$json_data")
+    highest=$(echo "$download_id" | tr ' ' '\n' | sort -n | tail -n 1)
+    echo "$highest"
+}
```

Comment:
```suggestion
get_download_id () {
  wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
  chmod +x ./jq
  cp jq /usr/bin
  json_data=$(curl -L --request GET --url 'https://www.tenable.com/downloads/api/v1/public/pages/nessus-agents/' --header 'Accept: aplication/json')
  download_id=$(jq --arg desc "$1" '.. | select(.description? == $desc) | .id' <<< "$json_data")
  highest=$(echo "$download_id" | tr ' ' '\n' | sort -n | tail -n 1)
  echo "$highest"
}
```

This is very pedantic - but can we normalise the indentation here?